README.md

```diff
@@ -94,4 +94,24 @@ At this point, you can increase or decrease the load in the LoadGenerator consol
 
 ## Deploying the application to Linux
 
-The script `deploy.sh` will build and deploy the application to a Linux machine using the `scp` command. Update the `DEPLOY_SERVER` variable at the top of the script to match your environment. Other changes may be required to this script to ensure the `scp` command has the proper permissions for your environment.
\ No newline at end of file
+The script `deploy.sh` will build and deploy the application to a Linux machine using the `scp` command. Update the `DEPLOY_SERVER` variable at the top of the script to match your environment. Other changes may be required to this script to ensure the `scp` command has the proper permissions for your environment.
+
+## Bridging with Azure (optional)
+
+NOTE: This section is optional
+
+`Warehouse.Azure` is a separate MVC application that will add or remove stock. It is used to demonstrate how to integrate with an application developed by another team in your organization. In this scenario, the external team uses an Azure Storage Queue and sends `ItemStockUpdated` events to it. Even though the Warehouse.Azure team maintains its own database and infrastructure, we want to be notified about changes in stock so we can update our EShop application accordingly.
+
+To set up the Warehouse.Azure project:
+
+1. Create a queue in Azure Storage
```

Comment:
Instead, how about:
Set up the proper resources needed for Azure Storage Queues in the Azure portal, so you can use a proper connection string.